File: cpp/a006.cpp
//一元二次方程式
#include <bits/stdc++.h>
using namespace std;
int main()
{
    int a, b, c, x, x1, x2, an;
    cin >> a >> b >> c;
    if (pow(b, 2) - 4 * a * c > 0)
    {
        x1 = ((-b) + sqrt((pow(b, 2)) - (4 * a * c))) / (2 * a);
        x2 = ((-b) - sqrt((pow(b, 2)) - (4 * a * c))) / (2 * a);
        cout << "Two different roots x1=" << x1 << " , x2=" << x2;
    }
    else if (pow(b, 2) - 4 * a * c == 0)
    {
        x = ((-b) + sqrt((pow(b, 2)) - (4 * a * c))) / (2 * a);
        cout << "Two same roots x=" << x;
    }
    else if (pow(b, 2) - 4 * a * c < 0)
    {
        cout << "No real root";
    }
    return 0;
}

File: Python/20230803/a006.py
a, b, c = map(int, input().split())
number = b ** 2 - 4 * a * c
if number > 0:
    ans1 = int((-b + number ** 0.5) / (2 * a))
    ans2 = int((-b - number ** 0.5) / (2 * a))
    print(f"Two different roots x1={ans1} , x2={ans2}")
elif number == 0:
    ans = int((-b + number ** 0.5) / (2 * a))
    print(f"Two same roots x={ans}")
else:
    print("No real root")
